udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearch.vue

```diff
@@ -23,11 +24,12 @@
       @blur="handleFocusOut"
     />
     <div
-      class="fr-collapse autocomplete w-100 fr-mt-1w shadow"
+      class="fr-collapse autocomplete w-100 fr-mt-1w shadow border-bottom-rounded"
```

Comment:
I don't find any `autocomplete` class. Should it be removed ? Or do I miss it ?

Reply:
the `fr-collapse autocomplete` is took from the Search component to keep uniformity, it apparently allows the collapse to fit the size of the searchbar